### PR TITLE
docs: add logo and favicon, update colors, copyright and authors

### PR DESCRIPTION
**Pull request by umarcor**
Similarly to ~SymbiFlow/symbiflow-docs#519~ fpga.org, this PR adds a logo and favicon to the docs, so it's easier to identify the tab/page.

There is a difference with regard to the colour of the logo. In ~symbiflow-docs~ fpga.org, it's blue because it's shown on a white background. However, the theme used here is slightly different, and it's shown on a ~purple~ indigo background. Therefore, the logo here is white.

~In the future, when/if the theme is updated, changing the SVGs should suffice.~

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -78,22 +82,25 @@
 # so a file named "default.css" will overwrite the builtin "default.css".
 # html_static_path = ['_static']
 
+html_logo = str(ROOT / 'images/logo.svg')
+html_favicon = str(ROOT / 'images/favicon.svg')
```
Comment (mithro) [inline docs/conf.py:86]: https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_favicon says it should be relative the configuration file and/or stored in _static?

Comment (umarcor) [reply docs/conf.py:86]: Agree. However, `_static` is not used in this repo. Shall I move `docs/images` to `docs/_static/images`?

See https://github.com/hdl/containers/tree/main/doc/_static.

Comment (umarcor) [reply docs/conf.py]: Changed in https://github.com/SymbiFlow/symbiflow-examples/pull/253/commits/801bfe59fe5b6ba2ab666733c281452d140ab87a.

**Comment by umarcor** [comment]
This PR now updates the colors and the copyright/authors (to match #249).

**Comment by umarcor** [comment]
@kgugala this is now updated to use F4PGA's logo and favicon.

When pushing it, GitHub Actions was triggered, but RTD did not run. Maybe "building PRs" needs to be enabled in https://readthedocs.org/projects/f4pga-examples/ ?

**Comment by kgugala** [comment]
> @kgugala this is now updated to use F4PGA's logo and favicon.
> 
> When pushing it, GitHub Actions was triggered, but RTD did not run. Maybe "building PRs" needs to be enabled in https://readthedocs.org/projects/f4pga-examples/ ?

I enabled this

**Pull request merged**